src/components/FilmDetailsPage/FilmDetailsPage.jsx

```diff
@@ -1,69 +1,55 @@
-import { useEffect, useState } from 'react';
-import { useParams } from 'react-router-dom';
+import { useSelector } from 'react-redux';
+import style from './FilmDetailsPage.module.css';
 import Header from '../Header';
 import LoadingPage from '../LoadingPage';
 import { translateGenre } from '../Helpers/translateGenre.js';
-import style from './FilmDetailsPage.module.css';
-
-const FilmDetailsSubscription = ( { property, value } ) =>
-{
-    return (
-        <>
-            <div className={style.detailItem}>
-                <p className={style.detailProperty}>{property}: </p>
-                <p>{value}</p>
-            </div>
-        </>
-    )
-}
-
-const FilmDetailsPage = () =>
-{
-    const { id } = useParams();
-    const [details, setDetails] = useState( null );
+import { FilmDetailsSubscription } from '../Helpers/filmDetailsSubscription.js';
+import AddToWatchListButton from '../AddToWatchListButton';
 
-    useEffect( () =>
-    {
-        const getFilmDetails = async () =>
-        {
-            const response = await fetch(
-                `https://api.themoviedb.org/3/movie/${id}?api_key=4d0c68776909a3f926088d7ddf14c097`,
-            );
-            const data = await response.json();
-            console.log( data );
-            setDetails( data );
-        };
-        getFilmDetails();
-    }, [id] );
 
+const FilmDetailsPage = () => {
+    const filmDetails = useSelector(state => state.filmApi.filmDetails)
+    const loading = useSelector(state => state.filmApi.loading)
+    function checkInLocalStorage(id) {
+        const localStorageWatchList = window.localStorage.getItem('watchList')
```

Comment:
First variable is JSON and second already parsed JSON :)

Reply:
Resolve it